Could not run the ResNet-50 example of creating the standalone bundles

This is my log, after running `make run`:
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

image-classifier: for the -model_input_name option: must be specified at least once!
Makefile:43: recipe for target 'build/resnet50.o' failed
make: *** [build/resnet50.o] Error 1
```

After adding `-model_input_name=gpu_0/data ` to line 44 in the make file, I get 
```
File ?resnet50/predict_net.pbtxt? already there; not retrieving.

File ?resnet50/predict_net.pb? already there; not retrieving.

File ?resnet50/init_net.pb? already there; not retrieving.

Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/cat_285.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/dog_207.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Allocated weights of size: 102121984
Expected weights of size: 102121984
Loaded weights of size: 102121984 from the file resnet50.weights
Allocated mutable weight variables of size: 606144
Loaded image: /datadrive/bhaskar/renesasai/glow/tests/images/imagenet/zebra_340.png
Loaded images size in bytes is: 602112
Copying image data into mutable weight vars: 602112 bytes
Expected to find variable 'output'
resnet50: resnet50.cpp:267: const SymbolTableEntry &getMutableWeightVar(const BundleConfig &, const char *): Assertion `mutableWeightVar && "Expected to find a mutable weight variable"' failed.
Aborted (core dumped)
Makefile:22: recipe for target 'run' failed
make: *** [run] Error 134
```

Can someone point out if I have to make any more changes to get it running?

--- a/examples/bundles/resnet50/resnet50.cpp
+++ b/examples/bundles/resnet50/resnet50.cpp
@@ -326,7 +326,8 @@ static uint8_t *allocateMutableWeightVars(const BundleConfig &config) {
 /// finding the index of the max element.
 static void dumpInferenceResults(const BundleConfig &config,
                                  uint8_t *mutableWeightVars) {
-  const SymbolTableEntry &outputWeights = getMutableWeightVar(config, "output");
+  const SymbolTableEntry &outputWeights =
+      getMutableWeightVar(config, "save_gpu_0_softmax");
   int maxIdx = 0;
   float maxValue = 0;
   float *results = (float *)(mutableWeightVars + outputWeights.offset);

